src/views/AskQuestion.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import graphql from 'babel-plugin-relay/macro'
+import { useHistory } from 'react-router-dom'
+
+import useForm from '../hooks/useForm'
+import { Button, Text } from '../styles'
+import FullPageForm from '../components/ui/FullPageForm'
+import TextInput from '../components/ui/TextInput'
+
+const mutation = graphql`
+  mutation AskQuestionMutation($input: QuestionCreateInput!){
+    questionCreate(input: $input) {
+      question {
+        id
+      }
+    }
+  }
+`
+
+function AskQuestion () {
```

Comment:
I'm letting the backend handle authentication validation. Since all error handling is standardized now, you'll see a message that you don't have permissions. Maybe in the future, we can build out some front-end logic, but I feel this is quite good for a start.